AutoInstaller.ps1

```diff
@@ -0,0 +1,38 @@
+#Requires -RunAsAdministrator
+
+# Check user has webhook URL ready
+$userPrompt = Read-Host -Prompt "Do you have your webhook URL ready? Y/N"
+if ($userprompt -ne 'Y') {
+    Write-Output "Please create a Discord webhook before continuing. `nFull instructions avalible at https://support.discord.com/hc/en-us/articles/228383668-Intro-to-Webhooks"
+    exit
+}
+# Get latest release from GitHub
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+$latestRelease = Invoke-WebRequest -Uri https://github.com/tigattack/VeeamDiscordNotifications/releases/latest -Headers @{"Accept"="application/json"} -UseBasicParsing
+# Release IDs are returned in a format of {"id":3622206,"tag_name":"v1.0"} so we need to extract tag_name.
+$latestVersion = $latestRelease.Content | ConvertFrom-Json | Select-Object tag_name -ExpandProperty tag_name

```

Comment:
This one's on me since I told you to do it like this before, but I have a new suggestion now. Sorry 😄

Re-using code isn't great practice if it's avoidable, so I suggest that this section of code be ripped out of both `AutoInstaller.ps1` and `UpdateVeeamDiscordNotification.ps1` and put into a module (something like `Get-LatestRelease.psm1`) in the `resources` dir.

This new module can then be imported in both aforementioned scripts and utilised in a much simpler fashion.